ros_ethercat_hardware/include/ros_ethercat_hardware/ethercat_hardware.h

```diff
@@ -259,6 +259,13 @@ class EthercatHardware
 
   hardware_interface::HardwareInterface *hw_;
 
+  // Attributes required to run multi-threaded calls to HardwareInterface.update (for multiple HardwareInterface's)
+  boost::mutex update_mutex;
+  boost::condition_variable start_of_work_condition_eth_hw_read;
```

Comment:
Is there a reason why you're using boost for all this instead of the std library?

Reply:
I'm OK with merging, but in general I think it's preferred to use std instead of boost for those things that exist in the std library.
We can discuss this with the team, as I'm not sure that this is really a rule.

Reply:
I did consider using std, but decided to stick with boost for consistency since that's what we are using in the rest of the code.

Reply:
Do you prefer that I change this one?

Reply:
E.g. here: https://github.com/shadow-robot/ros_ethercat/blob/51894e5cf60c1e908857b365d13276b283481938/ros_ethercat_hardware/include/ros_ethercat_hardware/ethercat_hardware.h#L172
and here:
https://github.com/shadow-robot/ros_ethercat/blob/51894e5cf60c1e908857b365d13276b283481938/ros_ethercat_model/include/ros_ethercat_model/ros_ethercat.hpp#L640

Reply:
Thing is that this is code that is 15 years old. I don't know if there were condition variables in the std library back then.
I'm OK with leaving it as it is. But I think we should agree on this at a team level and make it one of our style guidelines.